problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -1,17 +1,94 @@
 \begin{exercises}
+	% Topics
+	% formal + intuitive definition of subspace
+	% relationship between subspace and spans
+	% prove a set is a subspace
+	% basis + dimension of a subspace
 	\begin{problist}
-		\prob  Example problem with parts
+
+		\prob For each of the following sets, prove whether or not it is a subpace.
+		\begin{enumerate}
+			\item Let $\mathcal T \subseteq \R^2$ be the complete solution to $3x-y=0$.
+			\item Let $\mathcal U \subseteq \R^2$ be the complete solution to $\frac{1}{2}x-6y=0$.
+			\item Let $\mathcal V \subseteq \R^2$ be the complete solution to $x-5y-1=0$.
+			\item Let $\mathcal W \subseteq \R^3$ be the complete solution to $x+2y-3z=0$.
+			\item Let $\mathcal X \subseteq \R^3$ be the complete solution to
+			$5x-\pi y + (\ln 2)z=0$.
+			\item Let $\mathcal Q \subseteq \R^n$ be the complete solution to
+			$a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$ where $a_1,...,a_n \in \R$.
+		\end{enumerate}
+
+		\prob For each of the following sets, prove whether or not it is a subpace.
+		\begin{enumerate}
+			\item Let $\mathcal B \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{5\\-7}+\mat{1\\2}$
+			\item Let $\mathcal A \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{-3\\4}$
+			\item Let $\mathcal C \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{1\\0\\5}$
+			\item Let $\mathcal D \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{2\\3\\4}+s\mat{10\\20\\131}+\mat{0\\0\\6}$
+			\item Let $\mathcal E \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{5\\7\\1}+s\mat{2\\-2\\1}$
+		\end{enumerate}
+
+		\prob Use the definition of subspace to show each span below is a subspace.
+		\begin{enumerate}
+			\item $\Span\Set*{\mat{0\\1}, \mat{1\\2}}$
+			\item $\Span\Set*{\mat{1\\1\\1}, \mat{1\\0\\0}, \mat{2\\0\\0}}$
+		\end{enumerate}
+
+		\prob For each subspace (i) find its dimension and (ii) find two bases.
+		\begin{enumerate}
+			\item $\Span\Set*{\mat{2\\3}, \mat{-4\\-6}, \mat{1\\\frac{3}{2}}}$
+			\item $\Span\Set*{\mat{1\\0\\-2}, \mat{0\\2\\5}, \mat{1\\2\\3}}$
+			\item $\vec x = t\mat{6\\1\\1}+s\mat{6\\0\\6}$
```

Comment:
For these, you need to say something like, "the plane given in vector form by..."